app/views/admin/editions/show/_main.html.erb

```diff
@@ -0,0 +1,277 @@
+
+    <% if @edition.has_legacy_tags? %>
+      <% if @edition.has_primary_sector? %>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Primary specialist topics",
+          font_size: "m",
+          heading_level: 3,
+          margin_bottom: 3,
+        } %>
+        <%= render "govuk_publishing_components/components/list", {
+          visible_counters: true,
+          items: [specialist_sector_name(@edition.primary_specialist_sector_tag)]
+        } %>
+      <% end %>
+      <% if @edition.has_secondary_sectors? %>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Secondary specialist topics",
+          font_size: "m",
+          heading_level: 3,
+          margin_bottom: 3,
+        } %>
+        <%= render "govuk_publishing_components/components/list", {
+          visible_counters: true,
+          items: specialist_sector_names(@edition.secondary_specialist_sector_tags)
+        } %>
+      <% end %>
+    <% else %>
+      <p class="govuk-body">No specialist topic tags for this document</p>
+    <% end %>
+  </section>
+
+  <% if @edition.allows_attachments? %>
+    <section class="app-view-edition-summary__section app-view-edition-summary__section--attachments">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Attachments",
+        heading_level: 2,
+        font_size: "l",
+        margin_bottom: 3,
+      } %>
+
+      <p class="govuk-body">
+        <%= link_to("#{@edition.attachments.any? ? "Modify" : "Add"} attachments",
+                    admin_edition_attachments_path(@edition.id), class: 'govuk-link') %>
+      </p>
+
+      <% if @edition.attachments.any? %>
+        <%= render "govuk_publishing_components/components/table", {
+          head: [
+            {
+              text: "Title"
+            },
+            {
+              text: "Filename"
+            },
+            @edition.published? ? {
+              text: "Actions"
+            } : nil
+          ].compact,
+          rows: @edition.attachments.map do | attachment |
+            [
+              { text: attachment[:title] },
+              { text: link_to_attachment(attachment, preview: !@edition.published?, full_url: true, class: "govuk-link") },
+              @edition.published? ? { text: link_to("Edit", edit_admin_edition_attachment_path(@edition.id, attachment[:id]), class: "govuk-link") } : nil,
+            ].compact
+          end
+        } %>
+      <% else %>
+        <p class="govuk-body">No attachments for this document</p>
+      <% end %>
+    </section>
+  <% end %>
+  <% if @edition.translatable? %>
+    <section class="app-view-edition-summary__section">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Translations",
+        heading_level: 2,
+        font_size: "l",
+        margin_bottom: 3,
+      } %>
+      <% if @edition.editable? and @edition.missing_translations.any? %>
```

Comment:
Nit - I know this was the behaviour before, but i think we should use && unless theres a good reason not to throughout the codebase.

There's actually subtly different behaviour too which isn't obvious at a first glance https://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby

I know it doesn't directly affect this, bit i still think its better to just stick with `&&` unless there's a good reason not to.

Reply:
Thanks!